docs/_source/guides/llms/practical_guides/create_dataset.md

```diff
@@ -137,7 +137,8 @@ The next step is to create records following Argilla's `FeedbackRecord` format.
 - `fields`: A dictionary with the name (key) and content (value) of each of the fields in the record. These will need to match the fields set up in the dataset configuration (see [Define record fields](#define-record-fields)).
 - `metadata` (optional): A dictionary with the metadata of the record. This can include any information about the record that is not part of the fields. For example, the source of the record or the date it was created. If there is no metadata, this will be `None`.
 - `external_id` (optional): An ID of the record defined by the user. If there is no external ID, this will be `None`.
-- `responses` (optional): A list of all responses to a record. There is no need to configure this when creating a record, it will be filled automatically with the responses collected from the Argilla UI.
+- `suggestions`(optional): A list of all suggested responses for a record e.g., model predictions or other helpful hints for the annotators. These will need to follow the same schema as the responses of the type of question answered by the suggestion. If these are added, suggestions will appear in the UI as pre-filled responses.
+- `responses` (optional): A list of all responses to a record. You will only need to add them if your dataset already has some annotated records. Make sure that the responses adhere to the same format as Argilla's output and meet the schema requirements for the specific type of question being answered.
```

Comment:
```suggestion
- `responses` (optional): A list of all responses to a record. You will only need to add them if your dataset already has some annotated records. Make sure that the responses adhere to the same format as Argilla's output and meet the schema requirements for the specific type of question being answered. Also make sure to include the `user_id`s in case you're planning to add more than one response for the same question, as just one `user_id` can be None, later to be replaced by the current active `user_id`, while the rest will be discarded otherwise.
```